### ISSUE TITLE
OAuth 2.0 service authentication

### ISSUE THREAD

**Issue by fanfuvac** [provectus/kafka-ui#4500]
### Issue submitter TODO list

- [X] I've looked up my issue in [FAQ](https://docs.kafka-ui.provectus.io/faq/common-problems)
- [X] I've searched for an already existing issues [here](https://github.com/provectus/kafka-ui/issues)
- [X] I've tried running `master`-labeled docker image and the issue still persists there
- [X] I'm running a supported version of the application which is listed [here](https://github.com/provectus/kafka-ui/blob/master/SECURITY.md)

### Describe the bug (actual behavior)

We have configured OAuth 2.0 on kafka-ui, user login works correctly, but I am trying to implement authentication for our server to call API in an unattended way, and I am trying to use bearer token authentication, I am able to obtain client_credentials from our oauth server, but when I use this as a bearer token Authorization header for Kafka-ui, it redirects me to oauth login screen instead of returning the api content:
![image](https://github.com/user-attachments/assets/5c4d709b-81fb-41b6-9f72-6469443e2b87)

Is there some other way how to programatically gain access to API with oauth enabled?

### Expected behavior

Service would be granted access when providing bearer token from OAuth.

### Your installation details

Version: [83b5a60](https://github.com/provectus/kafka-ui/commit/83b5a60) v0.7.2

### Steps to reproduce

Sent request with this parameter to OAuth provider:
![image](https://github.com/user-attachments/assets/49157767-59bd-40e2-936d-2c5a8a6e8cd3)
Use clientid and Secret as authentication for this request
Get access_token as a response:
![image](https://github.com/user-attachments/assets/f82b1635-765e-48b0-9038-e76c7f24b0cf)
Use this Access token as describe in the description
Kafka-ui redirects me to login screen instead of giving me access.

**Comment by denouche** [comment]
Hello @fanfuvac , I have the same issue today, did you find a workaround?

**Comment by fanfuvac** [comment]
No, as far as I know, there is no workaround. Also, this project is no longer supported, I have opened an issue in its successor: https://github.com/kafbat/kafka-ui/issues/659 The issue is registered however it does not seems to get any traction.
